### DISCUSSION TITLE
How to obtain calculated version from build

### DISCUSSION THREAD

**Discussion by Crossbow78** [adamralph/minver#563] [Q&A] [answered]
I've added the MinVer nuget package to my .NET Core projects, and during build the version number is properly determined and gets embedded into the output assembly.

How would I be able to obtain the generated version tag in the remainder of my (Azure) pipeline, so I can use it eg. to correctly label the pipeline run? In other words, is there a way to get the value of the custom build [properties set by MinVer](https://github.com/adamralph/minver#version-numbers) outside that build process?

**Comment by adamralph** [comment, marked as answer]
@Crossbow78 please see the [FAQ](https://github.com/adamralph/minver#can-i-use-minver-to-version-software-which-is-not-built-using-a-net-sdk-style-project).
